org.eclipse.draw2d/src/org/eclipse/draw2d/graph/CompoundBreakCycles.java

```diff
@@ -29,38 +29,34 @@ class CompoundBreakCycles extends GraphVisitor {
 	 * from the list.
 	 */
 	private NodeList graphNodes;
-	private NodeList sL = new NodeList();
+	private final NodeList sL = new NodeList();
 
-	private boolean allFlagged(NodeList nodes) {
-		for (int i = 0; i < nodes.size(); i++) {
-			if (nodes.getNode(i).flag == false)
-				return false;
-		}
-		return true;
+	private static boolean allFlagged(NodeList nodes) {
+		return nodes.stream().allMatch(n -> n.flag);
 	}
 
-	private int buildNestingTreeIndices(NodeList nodes, int base) {
-		for (int i = 0; i < nodes.size(); i++) {
-			Node node = (Node) nodes.get(i);
-			if (node instanceof Subgraph) {
-				Subgraph s = (Subgraph) node;
+	private static int buildNestingTreeIndices(NodeList nodes, int base) {
+		for (Node node2 : nodes) {
+			Node node = node2;
```

Comment:
My apologies that I wasn't clear enough. The auto clean-up messed up this method here as well :sweat_smile:

Reply:
I got that from your comment but I thought I fixed it in my latest commit. Now I'm confused. Let me check

Reply:
I'm a dumbass. Forgot to push :facepalm:

Reply:
Many thx for your time and patience.

Reply:
No need to mention it :)